Sample Solutions/Data Lake/MGDC/src/JsonToCDM_CSV_SynapseNotebook.ipynb

```diff
@@ -0,0 +1,404 @@
+{
+  "cells": [
+    {
+      "cell_type": "code",
+      "source": [
+        "# # #Initializing for test purposes\r\n",
+        "\r\n",
+        "# Define the variables used for creating connection strings\r\n",
+        "# initialize the blank variables with correct values\r\n",
+        "# Application (Client) ID\r\n",
+        "# AppId = \"\"\r\n",
+        "# # Application (Client) Secret Key\r\n",
+        "# AppKey = \"\"\r\n",
+        "# # Directory (Tenant) ID\r\n",
+        "# TenantId = \"\"\r\n",
+        "# StorageAccountName = \"\"\r\n",
+        "# PipelineId = \"\"\r\n",
+        "\r\n",
+        "AppId = \"e5217431-1131-41ef-8946-488cd0fae886\"\r\n",
+        "# Application (Client) Secret Key\r\n",
+        "AppKey = \".Bj7Q~pSloqDMckFBxm1Cfgdaz9KbE0wVd6o0\"\r\n",
+        "# Directory (Tenant) ID\r\n",
+        "TenantId = \"bd382938-3ba0-469f-8b45-fa06740d181f\"\r\n",
+        "StorageAccountName = \"mgdcvivadatalake\"\r\n",
+        "PipelineId = \"93d30026-56bb-4401-bf83-76d91521f404\"\r\n",
+        "\r\n",
```

Comment:
There seems to be secrets here?  If they are active, should look at resetting them and remove the information from the notebook

Reply:
oops. Will do